app/app.go

```diff
@@ -591,8 +609,8 @@ func NewOsmosisApp(
 		// that in-memory capabilities get regenerated on app restart.
 		// Note that since this reads from the store, we can only perform it when
 		// `loadLatest` is set to true.
-		ctx := app.BaseApp.NewUncachedContext(true, tmproto.Header{})
-		app.CapabilityKeeper.InitializeAndSeal(ctx)
+		// ctx := app.BaseApp.NewUncachedContext(true, tmproto.Header{})
+		app.CapabilityKeeper.Seal()
```

Comment:
TODO: Double check

Reply:
The can be moved out of the loadLatest condition.

Reply:
Done, following gaia / regen's example